steps/deploy-to-itunes-connect-with-pilot/1.0.0/step.yml

```diff
@@ -0,0 +1,221 @@
+title: Deploy to TestFlight (with Pilot)
+summary: Pilot makes it easier to manage your app on Apple’s TestFlight.
+description: |-
+  Deploy & Distribute to TestFlight, using the Faslane Pilot
+  (https://github.com/fastlane/fastlane/tree/master/pilot)
+  Ruby gem.
+
+  **If this is the very first build** of the app
+  you try to deploy to iTunes Connect / TestFlight then you might want to upload the first
+  build manually to make sure it fulfills the initial iTunes Connect / TestFlight submission
+  verification process.
+
+  **Be advised** that this
+  step uses a well maintained, open source tool which
+  uses *undocumented and unsupported APIs* (because the current
+  TestFlight platform does not have a documented and supported API)
+  to perform the deployment.
+  This means that when the API changes
+  **this step might fail until the tool is updated**.
+
+  ## Setup guide and notes
+
+  * Register an app on iTunes Connect / TestFlight, on the **My Apps** page. Click on the *plus sign* and select the **New App** option. This requires an **admin** account.
+  * Requires an **App Store iOS distibution Provisioning Profile**
+  * Every build which you want to push to iTunes Connect / TestFlight have to have a **unique build and version number** pair (*increment either or both before a new deploy to iTunes Connect*)
+website: https://github.com/e-vision/steps-itunesconnect-pilot
+source_code_url: https://github.com/e-vision/steps-itunesconnect-pilot
+support_url: https://github.com/e-vision/steps-itunesconnect-pilot/issues
+published_at: 2018-02-14T16:09:14.814259+01:00
+source:
+  git: https://github.com/e-vision/steps-itunesconnect-pilot.git
+  commit: 0373588ae0444c56518b1ae14f27a576b259249f
+host_os_tags:
+- osx-10.10
+project_type_tags:
+- ios
+- macos
+- xamarin
+- cordova
+- ionic
+- react-native
+type_tags:
+- deploy
+toolkit:
+  go:
+    package_name: github.com/e-vision/steps-itunesconnect-pilot
+deps:
+  brew:
+  - name: go
+  apt_get:
+  - name: golang
+    bin_name: go
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+inputs:
+- ipa_path: $BITRISE_IPA_PATH
+  opts:
+    description: Path of the IPA file, to be deployed to iTunes Connect
+    summary: ""
+    title: IPA path
+- opts:
+    description: Path of the PKG file, to be deployed to iTunes Connect
+    summary: ""
+    title: PKG path
+  pkg_path: $BITRISE_PKG_PATH
+- opts:
+    description: 'Action to execute: build or distribute'
+    summary: ""
+    title: Pilot action
+  pilot_action: distribute
```

Comment:
I think this step is only supposed to distribute an app (based on its title).
Also based on `fastlane pilot -h` it seems there is no `build` command.
I would recommend to remove this input and hardcode the `distribute` action, or am i missing something?